torch/onnx/symbolic_helper.py

```diff
@@ -461,6 +461,11 @@ def _set_operator_export_type(operator_export_type):
     global _operator_export_type
     _operator_export_type = operator_export_type
 
+_training_mode = None
+def _set_training_mode(training_mode):
+    global _training_mode
```

Comment:
Do we have to make this global. Can we scope it more to a file and then import it where ever necessary?